Documentation/tpv34.rst

```diff
@@ -0,0 +1,168 @@
+SCEC TPV34
+==========
+
+The TPV34 benchmark uses a right-lateral, planar, vertical, strike-slip fault 
+set in a half-space. The velocity structure is the actual 3D velocity structure 
+surrounding the Imperial Fault, as given by the SCEC Community Velocity Model
+`CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_.
+
+.. figure:: LatexFigures/tpv34_imperialfault.png
+   :alt: The Imerial Fault.
+   :width: 15.00000cm
+
+   The Imperial Fault. The red line marks the Imperial Fault. It straddles the California-Mexico border, south of the Salton Sea. The Imperial Fault is approximately 45 km long and 15 km deep, with a nearly vertical dip angle ranging from 81 to 90 degrees according to the SCEC Community Fault Model CFM-4.
+
+Geometry
+~~~~~~~~
+
+The model volume is a half-space. The fault is a vertical,
+planar, strike-slip fault. The fault reaches the Earth’s surface. Rupture is
+allowed within a rectangular area measuring 30000 m along-strike and
+15000 m down-dip.
+
+There is a circular nucleation zone on the fault surface. The hypocenter is 
+located 15 km from the left edge of the fault, at a depth of 7.5 km.
+
+.. figure:: LatexFigures/tpv34.png
+   :alt: Diagram of TPV34.
+   :width: 15.00000cm
+
+   Diagram of TPV34. 
+
+The geometry is generated with `Gmsh <https://gmsh.info/>`_. All the files 
+that are needed for the simulation are provided at 
+https://github.com/SeisSol/Examples/tree/master/tpv34.
+
+.. figure:: LatexFigures/tpv34_gmshhalf.png
+   :alt: Diagram of TPV34 vertical fault in Gmsh.
+   :width: 15.00000cm
+
+   Diagram of TPV34 vertical fault in Gmsh.
+
+Material
+~~~~~~~~
+
+To obtain the velocity structure for TPV34, we need to install and run 
+the SCEC Community Velocity Model `CVM-H <https://strike.scec.org/scecpedia/CVM-H>`_. 
+For TPV34, we are using CVM-H version 15.1.0 and we use 
+`ASAGI <https://github.com/TUM-I5/ASAGI>`_ to map related material properties 
+for a SeisSol simulation. Detailed explanations are provided at 
+https://github.com/SeisSol/Examples/blob/master/tpv34/generate_ASAGI_file.sh
+
```

Comment:
Add something like:
We generate 2 netcdf files of different spatial resolution to map more finely the 3D material properties close to the fault.
The domain of validity of each netcdf files read by ASAGI is defined by the yaml `tpv34-material.yaml` file: